docs/ecs_guide.md

```diff
@@ -136,33 +149,34 @@ To create an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_role
     }
     ```
 
-2. **Create the IAM role**
+2. **Create the IAM roles**
 
-    Use the `aws iam create-role` command to create the role:
+    Use the `aws iam create-role` command to create the roles that you will be using. The `ecsTaskExecutionRole` will be used by the worker to talk to make AWS API calls on your behalf when starting ecs tasks.
 
     ```bash
 
-    aws iam create-role \ 
+    aws iam create-role \
     --role-name ecsTaskExecutionRole \
     --assume-role-policy-document file://ecs-trust-policy.json
     ```
 
+ !!! tip
+    Depending on the requirements of your task, it is advised to create a [second role for your tasks](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html). This will contain permissions required while running the task. Because our sample flow doesn't interact with AWS you don't have to.
```

Comment:
```suggestion
    Depending on the requirements of your task, it is advised to create a [second role for your tasks](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html). This role will contain permissions required while running the task. Because our sample flow doesn't interact with AWS we don't need a second role.
```

Reply:
Maybe add something about this being needed when interacting with other AWS services that require other permissions.